### PR TITLE
Allow schema store to be specified for AvroTurf::Messaging

### PR DESCRIPTION
**Pull request by tjwp**
@jturkel any internal feedback before I submit this?

### CODE REVIEW

File: spec/messaging_spec.rb [hunk 1]
```diff
@@ -38,16 +38,34 @@
...
   end
 
-  it "allows specifying a reader's schema" do
-    message = { "full_name" => "John Doe" }
-    data = avro.encode(message, schema_name: "person")
-    expect(avro.decode(data, schema_name: "person")).to eq message
+  it_behaves_like "a schema store"
+
+  context "with a provided schema store" do
+    let(:schema_store) { AvroTurf::SchemaStore.new(path: "spec/schemas") }
```
Comment (jturkel) [inline spec/messaging_spec.rb]: Should the test validate that this `schema_store` is actually used?

**Comment by jturkel** [comment]
Just the one question

File: spec/messaging_spec.rb (hunk 1 above)
Comment (tjwp) [inline spec/messaging_spec.rb]: Done

**Comment by tjwp** [comment]
Closing to submit to original repo.

**Pull request closed without merging**